modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -339,6 +341,8 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		UserLoginDateComparator.getInstance(false);
 		UserScreenNameComparator.getInstance(false);
 		WorkflowMetricsSLADefinitionVersionIdComparator.getInstance(false);
+		SavedContentEntryClassNameIdComparator.getInstance(false);
```

Comment:
@victorhcunha Why are these sorted at the end in this file? They should just be together with the ones starting with "S".